.github/actions/browser/action.yml

```diff
@@ -0,0 +1,82 @@
+name: 'Run Test On Browser'
+description: 'Setup Browser and Run Test'
+inputs:
+  # Support one browser per action for now, because some packages has build script before test
+  # which nx cannot send params to karma test after build
+  browsers:
+    description: 'Browser name'
```

Comment:
Purpose to rename this to 'Target browser(s)' instead.

Reply:
Changed